lib/cc_plugin_imos/cc_plugin_imos/imos.py

```diff
@@ -1214,11 +1214,11 @@ def check_fill_value(self, dataset):
             if not hasattr(var, '_FillValue'):
                 continue
 
-            result = Result(BaseCheck.MEDIUM, True, ('var', name, '_FillValue'))
+            result = Result(BaseCheck.LOW, True, ('recommended', name, '_FillValue'))
             if is_numeric(type(var._FillValue)) and np.isnan(var._FillValue):
                 result.value = False
                 result.msgs = [
-                    "Attribute %s:_FillValue must have a real numeric value, not NaN" % name
+                    "We request that _FillValue attribures be set to a real numeric value, not NaN"
```

Comment:
How about "We recommend..." instead of "request" to make clearer that this is not critical in order to pass the check.

I can see that you already modified the result display to print "recommended" instead of "var".